### ISSUE TITLE
[P3.1][Sourcing][Accessibility][p13n dialog] Checkbox is not being read by screenreader while browsing the 'Columns' inside <fdp-table>'s p13n dialog

### ISSUE THREAD

**Issue by yixuan-liu** [SAP/fundamental-ngx#10868]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
The checkbox and its status should be announced correctly to the screenreader when accessed.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
- Angular 15
- F-NGX Platform: 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Please see the `Personalization Dialog` section under this example:
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/p13-dialog-table

![Screenshot 2023-11-01 at 5 09 32 PM](https://github.com/SAP/fundamental-ngx/assets/22454655/20c30a74-86fd-478a-b0d4-c49ccc374979)

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
UI type, information, properties of all UI elements should be conveyed correctly once to the screen reader users when accessed.

**Comment by droshev** [comment]
```
<ul role="list">
  <li role="listitem"></li>
</ul>
```

**Comment by yixuan-liu** [comment]
Hi team, just wanted to check in and see if there is a timeline of fixing this issue? Thx
